### ISSUE TITLE
Why not use softmax before cross entropy?

### ISSUE THREAD

**Issue by LJXLJXLJX** [clovaai/deep-text-recognition-benchmark#136]
https://github.com/clovaai/deep-text-recognition-benchmark/blob/24749cd1a3e373a9e3180aa60b6d87ec4d621511/train.py#L162

I think there should be softmax here

**Comment by ku21fan** [comment]
Hello, 

We use `torch.nn.CrossEntropyLoss` and [This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class](https://pytorch.org/docs/stable/nn.html#torch.nn.CrossEntropyLoss).

Best.
